src/sentry/static/sentry/app/components/avatar/styles.tsx

```diff
@@ -13,10 +11,13 @@ export const imageStyle = (props: ImageStyleProps) => css`
   top: 0px;
   left: 0px;
   border-radius: ${props.round ? '50%' : '3px'};
-  border: ${props.suggested ? `1px dashed ${theme.gray400}` : 'none'};
```

Comment:
hmmm does removing this also remove the dashed border from a suggested team assignee?

Reply:
it is moved to the parent so that the css filters do not apply to it

Reply:
oooh my bad for some reason I thought the baseavatar stuff only applied to user avatars.